create.ps1

```diff
@@ -168,53 +186,41 @@ try {
             } | ConvertTo-Json -Depth 10
 
             $splatParams = @{
-                Uri         = "$($config.BaseUrl)/events/hr/v1/worker.business-communication.email.change"
+                Uri         = "$($actionContext.Configuration.BaseUrl)/events/hr/v1/worker.business-communication.email.change"
                 Method      = 'POST'
                 Body        = $body
                 Headers     = $headers
                 Certificate = $certificate
                 ContentType = 'application/json'
             }
+
             if (-not($dryRun -eq $true)) {
```

Comment:
This variable is not being used witin the V2 connector. So, I think we should move the code nested within the `if` statement to the `Update-Correlate` switch section.